src/components/NcUserBubble/NcUserBubble.vue

```diff
@@ -158,10 +167,10 @@ export default {
 		url: {
 			type: String,
 			default: undefined,
-			validator: url => {
+			validator: (url) => {
 				try {
-					url = new URL(url)
-					return !!url
+					url = new URL(url, url?.startsWith?.('/') ? window.location.href : undefined)
```

Comment:
```suggestion
					url = new URL(url, url?.startsWith?.('/') ? window.location.origin : undefined)
```
Links are relative to the origin in my testing

Reply:
I do not think this is correct, e.g. Current page is `http://foo.bar/a/b.php` and the relative url is:
1. `/c`
2. `./c`

Using `origin` will result:
1. `http://foo.bar/c`
2. `http://foo.bar/c`

Using `href` will result:
1. `http://foo.bar/c`
2. `http://foo.bar/a/c`

So I would say `href` is correct as otherwise the relative URL `./c` would resolve to `root/c` instead of `/a/c`.

Reply:
I believe new URL() will work either way though the difference would be checking whether `url?.startsWith?.('/')` or `url?.startsWith?.('./')`, but in this case I don't think it matters

Reply:
Yes but I am not sure if `http://foo.bar/../c` is valid